package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "app-builder",
-  "version": "7.2.0-SNAPSHOT",
+  "version": "7.2.1-SNAPSHOT",
```

Comment:
you could rewrite version as 7.3.0-SNAPSHOT in this PR to not open another PR ...